src/Styleguide/Pages/Artwork/Sidebar/ExtraLinks.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import styled from "styled-components"
+import { Sans } from "@artsy/palette"
+import { space, SpaceProps } from "styled-system"
+
+export interface ExtraLinksProps {
+  artwork: {
+    readonly __id: string
+    readonly is_for_sale: boolean
+    artists: Array<{
+      readonly __id: string
+      readonly is_consignable: boolean
+    }>
+  }
+}
+
+const ExtraLinksContainer = styled.div.attrs<SpaceProps>({})`
+  ${space};
+`
+
+export class ExtraLinks extends React.Component<ExtraLinksProps> {
+  render() {
+    const isForSaleArtwork = this.props.artwork.is_for_sale
+    const consignableArtists = this.props.artwork.artists.filter(
+      artist => artist.is_consignable
+    )
+    if (!isForSaleArtwork && consignableArtists.length === 0) {
+      return null
+    }
+    return (
+      <ExtraLinksContainer pb={2}>
```

Comment:
If you just need a container you can use `Box`. It's a generic div w/ some styled-system hooks.